guides/contributing/Updating Changelogs.md

```diff
@@ -0,0 +1,65 @@
+
+## Changelog entries
+
+Each bullet point in the changelog is called an entry. It describes the change that has been introduced in the package. Here is what defines a good changelog entry:
+
+- It is descriptive and concise — explains well the change to a reader who has _zero context_ about the change. If you have trouble making it both concise and descriptive, err on the side of descriptive.
+- It is placed under `Unpublished` version and the appropriate category ([follow rules described here](#categories)).
+- It has links to PR's author and PR itself (`expo-ci` bot can suggest them for you, read more below in this section).
+- It may contain only text and links. Any other markdown elements (such as lists, headers, blockquotes, tables, images, inline HTML) are not permitted — if you need them to provide important tips for the users to adapt to the (breaking) change, you should put this in the PR's description or link to a separate document on [`expo/fyi`](https://github.com/expo/fyi).
+
+An example of the correct entry:
+
+> \- Added a guide about updating changelogs in the packages. ([#13075](https://github.com/expo/expo/pull/13075) by [@tsapeta](https://github.com/tsapeta))
+
+We would like each entry to contain links to the PR and author's profile, so that everyone can find out more details about that change by opening the PR's page or, as a last resort, contact the author.
+Moreover, when the package is about to be published, our automated script runs through these PRs to find closed issues and then comments on such issues to let subscribed users know that the fix is available to the public (usually as `next` tag on NPM).
+At first it might sound annoying to create the PR first (to get PR's number), update changelog and then push once again. However, in thanks to our code reviewing bot, you can skip the links at the end — it'll post code review suggestions to add them.
+
+## Categories
+
+We're sticking with a few types of changes which are grouped together within the category (Markdown section with a third-level heading). It's important to put the entries under correct category - the categories are also used in the publish script to help decide how to bump the package version. Here is a list of predefined categories used across all changelogs in the repository 👇
+
+- **🛠 Breaking changes**
+
+  For changes in the API that may require users to change their code or project settings. Implies **major** bump when publishing the package.
+
+- **🎉 New features**
+
+  For new features and non-breaking changes in the API. Implies at least **minor** bump when publishing the package.
+
+- **🐛 Bug fixes**
+
+  For bug fixes and changes to the documentation that clarify any ambiguities.
+
+- **⚠️ Notices**
+
+  For changes that don't fit into the above sections, but users should be aware of as they deprecate existing API while still keeping backwards compatibility or cause behavior changes in some corner cases.
```

Comment:
```suggestion
  For changes that don't fit into the above sections, but that users should be aware of, because they deprecate existing an API while still keeping backwards compatibility or cause behavior changes in some corner cases.
```